plutus-wallet-api/ledger/Ledger/Validation.hs

```diff
@@ -270,8 +262,11 @@ eqPubKey (PubKey (LedgerBytes l)) (PubKey (LedgerBytes r)) = P.equalsByteString
 {-# INLINABLE txSignedBy #-}
 -- | Check if a transaction was signed by the given public key.
 txSignedBy :: PendingTx -> PubKey -> Bool
-txSignedBy (PendingTx _ _ _ _ _ _ sigs hsh) k =
```

Comment:
I actually think arguably the original was nicer. In fact, my vote would probably be for `PendingTx{pendingTxSignatures=sigs, pendingTxHash=hash}`